cardano-cli/src/Cardano/CLI/EraBased/Run/Genesis.hs

```diff
@@ -560,13 +560,7 @@ runGenesisCreateStakedCmd
   forM_ [ 1 .. numUTxOKeys ] $ \index ->
     createUtxoKeys utxodir index
 
-  mayStakePoolRelays
-    <- forM mStakePoolRelaySpecFile $
-       \fp -> do
-         relaySpecJsonBs <-
-           handleIOExceptT (GenesisCmdStakePoolRelayFileError fp) (LBS.readFile fp)
-         firstExceptT (GenesisCmdStakePoolRelayJsonDecodeError fp)
-           . hoistEither $ Aeson.eitherDecode relaySpecJsonBs
+  mayStakePoolRelays <- forM mStakePoolRelaySpecFile TN.readRelays
```

Comment:
As in other parts of `create-staked`, it's `create-staked`'s implementation that calls `create-testnet-data`'s implementation when they are shared. So nothing new here.